packages/kotti-ui/source/kotti-user-menu/types.ts

```diff
@@ -1,19 +1,28 @@
-import { KottiNavbar } from '../kotti-navbar/types'
-import { SpecifyRequiredProps } from '../types/utilities'
+import { z } from 'zod'
 
 export namespace KottiUserMenu {
-	export type SectionLink = KottiNavbar.SectionLink
-	export type Section = KottiNavbar.Section
+	export const sectionLinkSchema = z
+		.object({
+			link: z.string().optional(),
+			title: z.string(),
+		})
+		.passthrough()
```

Comment:
I flagged theses as `passthrough` because we allow custom properties on them. Unfortunately, this didn’t fix the type definition though. Perhaps worth making an issue at `zod` for